live-build/build.gradle

```diff
@@ -22,11 +22,11 @@ task ancillaryRepository(type: Exec) {
     inputs.file "${rootProject.projectDir}/scripts/build-ancillary-repository.sh"
 
     for (envVar in ["AWS_S3_URI_VIRTUALIZATION",
-                    "AWS_S3_URI_LINUX_PKG",
+                    "AWS_S3_URI_USERLAND_PKGS",
                     "AWS_S3_URI_MASKING",
                     "AWS_S3_URI_ZFS",
```

Comment:
Should this be `KERNEL`? (i.e. where is the `connstat` package coming from?)

Reply:
In the first phase of the rollout we do not use the kernel list yet, so we leave connstat and delphix-kernel as part of the userland list.
TOOL-7381 will be responsible of cleaning up the userland list.